files/zh-cn/web/javascript/reference/global_objects/bigint/tolocalestring/index.html

```diff
@@ -57,7 +57,7 @@ <h3 id="Using_locales">Using <code>locales</code></h3>
 
 // the nu extension key requests a numbering system, e.g. Chinese decimal
 console.log(bigint.toLocaleString('zh-Hans-CN-u-nu-hanidec'));
-// → 一二三,四五六,七八九,一二三,四五六,七八九
+// → 一二三，四五六，七八九，一二三，四五六，七八九
```

Comment:
not right here, and I'm not sure such a case:

```md
这是一个指向维基百科的{{interwiki("wikipedia", "传输控制协议#TCP窗口缩放选项", "TCP 窗口缩放选项")}}。
```

Reply:
cloud just ignore lines begin with `/\s*\/\//` ?